src/collections/_documentation/workflow/alerts-notifications/alerts.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Alerts
+sidebar_order: 1
+---
+
+Sentry users with admin permissions or higher can create one of two types of alerts: [Metric Alerts](#metric-alerts) or [Issue Alerts](#issue-alerts). You can use [Metric Alerts](#metric-alerts) for high-level monitoring of patterns, or fine-grained monitoring of individual events. An [Issue Alert](#issue-alerts) fires whenever any issue in the project matches specified criteria, such as a resolved issue re-appearing or an issue affecting many users. To confirm or set admin permissions, see **Project Settings > Alerts > New Alert**. 
+
+## Metric Alerts
+
+{% capture __alert_content -%}
+Metric Alerts are currently in beta. For more information about access to Metric Alerts, feel free to reach out at alerting-feedback@sentry.io.
+{%- endcapture -%}
+{%- include components/alert.html
+    title="Note"
+    content=__alert_content
+    level="warning"
+%}
+
+Metric alerts allow you to filter and set thresholds on all errors across a project. Use metric alerts for high-level monitoring of patterns or fine-grained monitoring of individual events. Metric alerts help express lightweight service-level objectives (SLOs) such as users affected by signup-page errors or volume of database errors.
+
+Alternatively, [issue alerts]({%- link _documentation/workflow/alerts-notifications/alerts.md -%}#issue-alerts) notify you about the impact of a specific issue.
+
+### Alert Builder
+
+A metric alert has, at most, two triggers. The first is a critical trigger, which is required. The second is a warning trigger, which is optional. Triggers are independent of one another; however, the warning (optional) must be reached before the critical (required) trigger.
+
+Sentry evaluates triggers approximately every minute from the highest severity to lowest. Sentry creates an alert with the severity of the matched trigger (warning or critical). If an alert is already active, its status is updated. Admins can resolve alerts manually, or automatically by setting the resolution threshold.
+
+When an alert is created or changes status, the actions associated with the trigger are executed. The available actions are:
+
+- Send an email (to a member or team). If sent to a member, the member's personal project alert opt-out settings are overridden.
+- Send a Slack notification
+
+### Alert Stream
+
+Alerts can be accessed via the **Alerts** sidebar item.
+
+[{% asset notifications/alert-stream.png alt="Alert stream illustrates alert duration, trends, severity, time duration and percentage of users affected." %}]({% asset notifications/alert-stream.png @path %})
```

Comment:
This design might be slightly changing soon 😬

Reply:
When the design changes, will you or @adhiraj send me a new screenshot and I'll update?